src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVarselRepository.kt

```diff
@@ -92,8 +97,11 @@ class AktivitetskravVarselRepository(private val database: DatabaseInterface) {
 private object Queries {
     const val selectExpiredVarsler =
         """
-            SELECT *
-            FROM aktivitetskrav_varsel
+            SELECT a.personident, varsel.*
+            FROM aktivitetskrav_varsel varsel
+                INNER JOIN aktivitetskrav_vurdering vurdering
+                    ON varsel.aktivitetskrav_vurdering_id = vurdering.id
+                INNER JOIN aktivitetskrav a ON a.id = vurdering.aktivitetskrav_id                
             WHERE expired_varsel_published_at IS NULL
                 AND svarfrist <= (NOW() - INTERVAL '1 weeks')
```

Comment:
Husk å fikse denne i ny PR (ref avklaring på syfo-aktivitetskrav-slack)

Reply:
Yes! 🙌 [oppgave](https://trello.com/c/xB0QiUYy/1649-dato-for-svarfrist-for-innbygger-og-dato-veileder-skal-f%C3%A5-varsel-oppgaven-i-oversikten-skal-v%C3%A6re-den-samme)